application/src/main/java/uk/gov/hmcts/ccd/definition/store/ApplicationConfiguration.java

```diff
@@ -36,6 +36,20 @@ public Executor asyncExecutor(@Value("${async.executor.core.pool.size}") Integer
         return executor;
     }
 
+    @Bean("validateExecutor")
+    public Executor validateExecutor(@Value("${validate.executor.core.pool.size:8}") Integer corePoolSize,
+                                     @Value("${validate.executor.max.pool.size:8}") Integer maxPoolSize,
```

Comment:
better if we combine all default values into app properties - though it may not have been adhered to in few other places.

Reply:
default values were removed, as they are initialized via application.properties under any circumstances.